core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/api/convertTo.kt

```diff
@@ -252,22 +254,15 @@ internal fun AnyFrame.convertToImpl(
             }
         }.toMutableList()
 
-        // when the target is nullable but the source does not contain a column, fill it in with nulls / empty dataframes
+        // when the target is nullable but the source does not contain a column,
+        // fill it in with nulls / empty dataframes
         val size = this.size.nrow
         schema.columns.forEach { (name, targetColumn) ->
-            val isNullable =
-                // like value column of type Int?
-                targetColumn.nullable ||
-                    // like value column of type Int? (backup check)
-                    targetColumn.type.isMarkedNullable ||
-                    // like DataRow<Something?> for a group column (all columns in the group will be nullable)
-                    targetColumn.contentType?.isMarkedNullable == true ||
-                    // frame column can be filled with empty dataframes
-                    targetColumn.kind == ColumnKind.Frame
-
             if (name !in visited) {
-                newColumns += targetColumn.createEmptyColumn(name, size)
-                if (!isNullable) {
+                try {
+                    newColumns += targetColumn.createNullFilledColumn(name, size)
```

Comment:
Could we improve the logic in that place a little bit? I have a feeling that we need to log something in catch section, or use check()/error() for createNullFilledColumn if it's possible?

Reply:
Sure, that makes sense

Reply:
Though, a simple log of the exception should be enough, as any `missingPaths` left will inform the user with  an IllegalArgumentException in line 296